### ISSUE TITLE
[FEATURE REQUEST] Python over the net to .NET

### ISSUE THREAD

**Issue by hackabean** [byt3bl33d3r/SILENTTRINITY#125]
**---- READ THIS PLEASE ----**

If the issue you're about to open falls under the following categories, I'll exercise my internet given right to comment with a meme and will close it immediately:

- A feature request is not a bug, please only open an issue ticket if you encounter a bug. Ideally if you want X feature to be added you'd submit a pull request, otherwise join the #silenttrinity channel in the [BloodHoundGang](https://bloodhoundgang.herokuapp.com/) Slack and yell at me there.

- Being rude and/or implying that I should drop everything I'm doing right now to fix it: just don't, you're getting this for free and I have no obligation to do anything 🖕.

- "X gets detected by AV/EDR product": this is inevitable for an open-source C2 tool. You can however submit a pull request to fix it and I'll award you with a "dank af" label which is legit street cred.

-  Opening an issue without filling out the required information below and/or giving an insanely vague description of the error you encountered

- Not using GIthub Markdown to format errors/output etc.. , I don't want to go blind please make it look pretty

**---- REMOVE THIS BLOCK AFTER READING ----**

# Issue Template

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* SILENTTRINITY Version:
* OS running the Client:
* Python Version Running the Client
* OS running the TeamServer:
* Python version running the Teamserver:

## Expected Behavior

Please describe the behavior you are expecting.

## Current Behavior

What is the current behavior?

## Failure Information

Please help provide information about the failure if this is a bug.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. step 1
2. step 2
3. step 3
   ...

### Failure Logs

Please include any relevant log snippets or files here.

Hey, not really a bug but the feature request. I am happy to work on that if that is something you would like to implement into ST. In your spare time, take a look at my blog post:

https://rafalharazinski.gitbook.io/security/personal-projects/case-of-executing-python-projects-through-silenttrinity

**Comment by byt3bl33d3r** [comment]
IronPython was replaced with Boolang for a number of reasons, first and foremost being that Boolang natively supports PInvoke. I do think IronPython would be an awesome addition, but because of this it's not worth implementing currently, additionally I rather wait until IronPython 3 is done before even starting to tackle this.

**Comment by hackabean** [comment]
Yeah I think that adding python 2.7 functionality now would be less better :) than waiting for python 3.
